### ISSUE TITLE
Write VCD file

### ISSUE THREAD

**Issue by Martoni** [ics-jku/wal#30]
Is it possible to write in vcd file ?
Could be useful to annotate vcd traces.

**Comment by LucasKl** [comment]
No, not at the moment. Although you can add new signals to a trace [like here](https://github.com/ics-jku/wal/blob/main/examples/virtual-examples/virtual1.wal), and I have an experimental (still private) branch that allows Surfer to connect to WAL to show these signals.
So this is something we are working quite actively on.

I will have a bit more time starting next week and will try to implement writing back vcds, though.
